### ISSUE TITLE
Some questions about scheduled task

### ISSUE THREAD

**Issue by kaaass** [ForeverOpp/kindlerss#1]
About scheduled task, a simple virtual server won't allow users to modify "php.ini" or use shell. For php,  as u know, using `set_time_limit(0);` and `ignore_user_abort(true);` won't be a good choice. But the cron can be a good way, because cPane support it.

WHY not using python???

**Comment by Evyde** [comment]
Yes , I konw,and SAE support the cron.It is because python isn't supported by some hosts.So I think PHP is good in this thing.
